vector/src/main/java/im/vector/riotredesign/features/home/HomeActivity.kt

```diff
@@ -51,15 +53,18 @@ class HomeActivity : VectorBaseActivity(), ToolbarConfigurable {
     private val homeActivityViewModel: HomeActivityViewModel by viewModel()
     private val homeNavigator by inject<HomeNavigator>()
 
+    private var progress: ProgressDialog? = null
+
     private val drawerListener = object : DrawerLayout.SimpleDrawerListener() {
         override fun onDrawerStateChanged(newState: Int) {
             hideKeyboard()
         }
     }
 
+    override fun getLayoutRes() = R.layout.activity_home
```

Comment:
I don't like this kind of method, I think it doesn't bring much and force using a layout file...

Reply:
It's not mandatory to use it